reccmp/isledecomp/formats/mz.py

```diff
@@ -0,0 +1,74 @@
+from dataclasses import dataclass
+from pathlib import Path
+import struct
+
+from .image import Image
+
+
+class MZHeaderNotFoundError(ValueError):
+    """MZ magic string not found"""
+
+
+# pylint: disable=too-many-instance-attributes
+@dataclass(frozen=True)
+class ImageDosHeader:
+    # Order is significant!
+    e_magic: int
```

Comment:
Maybe `bytes` here to match PE? And change the struct string in `from_memory()` and `taste()` to match

Reply:
Yeah, probably.
The header has `USHORT` though, but that shouldn't stop us.
https://github.com/hughbe/windows-sdk-headers/blob/89f151d343587fcd5b854a4851fcb8f7187f3ac4/Include/10.0.19041.0/km/ntimage.h#L70